### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by findme8** [Imraj-Singh/Score-Based-Generative-Models-for-PET-Image-Reconstruction#3]
Hi, thanks for your contribution in this field. I have downloaded the datasets and tried to use them for some experiments. I want to know that if your dataset include 'sinogram' also? Thanks!

**Comment by Imraj-Singh** [comment]
Yes the sonogram is included, double check the keys but it could be "measurements"...
